src/rulesets/finder.ts

```diff
@@ -2,7 +2,8 @@ import * as path from '@stoplight/path';
 import * as fs from 'fs';
 import { RESOLVE_ALIASES, STATIC_ASSETS } from '../assets';
 
-const SPECTRAL_SRC_ROOT = path.join(__dirname, '..');
+// let's point at dist directory that has all relevant files (including custom functions) transpiled
+const SPECTRAL_SRC_ROOT = path.join(__dirname, '../../dist');
```

Comment:
@P0lip What led you to switch over to `dist`?

Reply:
`src` isn't the best content root, since it does not have all files transpiled, i.e. custom functions are left as typescript files.
At the time I wrote the code, we didn't have the concept of custom functions, so it didn't really matter since we stored JSON files in both `src` and `dist` directories.
Does it make sense?

Reply:
Perfect sense!